spec/features/welcome/index_spec.rb

```diff
@@ -5,16 +5,19 @@
     visit root_path
   end
 
-  it 'displays welcome message' do
-    expect(page).to have_content(@welcome_message)
-  end
+# TODO - define welcome_message before expectation
+  # it 'displays welcome message' do
+  #   expect(page).to have_content(@welcome_message)
+  # end
 
-  it 'displays application description' do
-    expect(page).to have_content(@description)
-  end
+# TODO - define description before expectation
+  # it 'displays application description' do
+  #   expect(page).to have_content(@description)
+  # end
 
   it 'displays login fields for username and password' do
-    page.has_field?('Username', type: 'username')
```

Comment:
If you run on your comp... and have it as expect(page).to have_field('username)... do you have issue? I typically use that syntax but it was failing across the board. Found this, and seemed ot have passed (alas, I was wrong). Thoughts?

Reply:
mine worked alright with that syntax...weird that it doesn't work for you!

Reply:
this test is not performing as expected though. Going to go look at it now.